x-pack/plugins/observability_solution/synthetics/server/alert_rules/tls_rule/message_utils.ts

```diff
@@ -144,12 +150,13 @@ export const setTLSRecoveredAlertsContext = async ({
       newStatus = previousStatus;
     }
 
-    alert.setContext({
+    const context = {
       ...state,
       newStatus,
       previousStatus,
       summary: newSummary,
       [ALERT_DETAILS_URL]: alertUrl,
-    });
+    };
+    alertsClient.setAlertData({ id: recoveredAlertId, context });
```

Comment:
Are there any unit/integration tests for this rule type? I think this is one of the few that use alert state

Reply:
There are no tests for the rule from what I understand. Do you think I should add some for the alert state?

Reply:
Either that or if the reviewer from the obs-ux team is able to verify manually generating and recovering an alert to ensure everything is 👍 . Ideally tho we'd have functional tests for all these rule types although I understand we don't own this one

Reply:
But maybe at least a unit test for this util function if it's straightforward?

Reply:
yeah sure!

Reply:
Resolved in this commit [1068617](https://github.com/elastic/kibana/pull/186551/commits/10686177bc6fc612581d951b54e62a696fd21c08)